astronomer/providers/google/cloud/sensors/gcs.py

```diff
@@ -78,3 +84,86 @@ def execute_complete(self, context, event=None):  # pylint: disable=unused-argum
             raise AirflowException(event["message"])
         self.log.info("File %s was found in bucket %s.", self.object, self.bucket)
         return event["message"]
+
+
+class GCSObjectsWithPrefixExistenceSensorAsync(BaseOperator):
```

Comment:
Inherit from `GCSObjectsWithPrefixExistenceSensor` class so that we can use the ui_color and other attributs

Reply:
changed to sync version and added polling interval in __init__